auto_nag/scripts/component.py

```diff
@@ -11,7 +11,13 @@
 
 
 class Component(BzCleaner):
-    def __init__(self):
+    def __init__(
+        self,
+        confidence_threshold: float = 0.35,
+        general_confidence_threshold: float = 0.8,
+        days_lookup: int = 365,
+        max_days_in_cache: int = 7
```

Comment:
In reference to https://github.com/mozilla/relman-auto-nag/issues/1437#issuecomment-1505276793, both `days_lookup` and `max_days_in_cache` are used outside the  `auto_nag/scripts` directory:

- https://github.com/mozilla/relman-auto-nag/blob/941e19a8a23a180bf8a5080c4dfed40537846049/auto_nag/bzcleaner.py#L82
- https://github.com/mozilla/relman-auto-nag/blob/941e19a8a23a180bf8a5080c4dfed40537846049/auto_nag/bzcleaner.py#L188

Refactoring these is a bit less straightforward, so let us keep it outside the scope of this pull request.

Reply:
Got it. Since they are used outside the directory, I should remove them from the init function, correct?

Reply:
yes